airflow-core/src/airflow/ui/public/i18n/locales/he/admin.json

```diff
@@ -21,11 +21,14 @@
     },
     "connection_one": "חיבור",
     "connection_other": "חיבורים",
+    "connection_two": "חיבורים",
     "delete": {
       "deleteConnection_one": "מחק חיבור אחד",
```

Comment:
Is the אחד really necessary? I'd be ok with יחיד\בודד if we must have a word there